extensions/firebase-import-export/src/firebase-import.tsx

```diff
@@ -0,0 +1,41 @@
+import { getPreferenceValues, showHUD, closeMainWindow, getSelectedFinderItems, confirmAlert } from "@raycast/api";
+import { initializeFirebaseApp, restore } from "firestore-export-import";
+import { IImportOptions } from "firestore-export-import/dist/helper";
+import { Preferences } from "./interfaces";
+import { readFile } from "./utils";
+
+/* Getting the preferences from the Raycast API. */
+const preferences = getPreferenceValues<Preferences>();
+
+/* Setting the export options. */
+const importOptions: IImportOptions = {
+  autoParseDates: true,
+  autoParseGeos: true,
+  showLogs: false,
+};
+
+/* Initializing the Firebase app with the service account. */
+initializeFirebaseApp(JSON.parse(readFile(preferences.firebaseAuth)));
+
+export default async function Command() {
+  try {
+    const fileSystemItems = await getSelectedFinderItems();
+    if (
+      await confirmAlert({ title: "🚨 Are you sure to restore ? It will override or add new data to the collection" })
```

Comment:
```suggestion
      await confirmAlert({ icon: "🚨", title: "Are you sure you want to restore these collections? This will override or add new data to the current collections" })
```